core/server/data/migrations/init/2-create-fixtures.js

```diff
@@ -10,6 +11,15 @@ module.exports = function insertFixtures(options) {
 
     return Promise.mapSeries(fixtures.models, function (model) {
         logging.info('Model: ' + model.name);
+
+        // The Post model fixtures need a `published_at` date, where at least the seconds
+        // are different, otherwise `prev_post` and `next_post` helpers won't workd with
+        // them.
+        if (model.name === 'Post') {
+            _.forEach(model.entries, function (post, index) {
+                post.published_at = moment().add(index, 'seconds');
```

Comment:
Yeah either as strings in the `fixtures.json` or handled model layer.

In my opinion, the model defines what is allowed and handles fallbacks and conditions in a centralised place. 

I can think of a valid case, that a public write access adds a post, which is published without a published date. But for now we can limit the `published_at` fallback for internal usage.